mozfun/bits28/active_in_range/metadata.yaml

```
@@ -1,2 +1,6 @@
-friendly_name: Bits28 Days Active in Range
-description: Return a boolean indicating if any bits are set in the specified range of a bit pattern.
+description: 'Return a boolean indicating if any bits are set in the specified range
+  of a bit pattern. The start_offset must be zero or a negative number indicating
```

```suggestion
  of a bit pattern. The `start_offset` must be zero or a negative number indicating
```

We should consistently use md code formatting for parameter names.